Amethyst/Managers/WindowManager.swift

```diff
@@ -13,6 +13,8 @@ import RxSwiftExt
 import Silica
 import SwiftyJSON
 
+private let mouseDragRaceThresholdSeconds = 0.15
```

Comment:
Global statics like this should be PascalCase not camelCase, but I think this works better inside the `MouseStateKeeper`.
```swift
class MouseStateKeeper {
    private let mouseDragRaceThresholdSeconds = 0.15
    ...
}
```